### PR TITLE
Restructure navbar&titles

### LINKED ISSUE #629: Restructure the nav bar / titles
### Problem
There are various improvements in the DF page that we should implement.
Let's restructure the Nav Bar + Pages a bit so they are easier to follow

### DoD:
- [x] Rename veOCEAN => PASSIVE
- [x] Move `Delegate` inside of `Volume-DF` (bottom of page)
- [x] Top Nav Bar should be: [PASSIVE, VOLUME DF, CHALLENGE DF, REWARDS, ABOUT]
- [ ] 1 title + 1 sentence description of how each page works

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #629.

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -225,11 +224,11 @@
     <Route path="/volume-df" primary={false}>
       <DataPortal />
     </Route>
-    <Route path="/veocean" primary={false}>
+    <Route path="/passive-df" primary={false}>
       <VeOceanPortal />
     </Route>
-    <Route path="/delegate" primary={false}>
-      <DelegationPortal />
```
Comment (idiom-bytes) [inline src/App.svelte:232]: Hi Norbert, please do not delete routes.
Simply update the route and push the user to the right place.

**Comment by idiom-bytes** [comment]
Hi Norbert, looks good. Some notes:
- Please, don't delete routes, just update them to send the user to the right place.
- Please, don't use `&` on the git commit name because it causes all sorts of issues :sweat_smile: 
- I see what you're doing with the subtitle in volume-df page, but we need to adjust it. I'm going to do a quick fix here, and then create a separate issue to adjust these things.

Screenshot from 2023-08-21 10-11-26
_Now the sentence doesn't describe the feature, and instead tells the user about the delegation feature_

**Review by idiom-bytes — APPROVED** [review]
Looks good.

**Review by idiom-bytes — APPROVED** [review]
lgtm

File: src/App.svelte [hunk 2]
```diff
@@ -227,7 +227,10 @@
     <Route path="/passive-df" primary={false}>
       <VeOceanPortal />
     </Route>
-     <Route path="/veocean" primary={false}>
+    <Route path="/delegate" primary={false}>
+      <VeOceanPortal />
```
Comment (KatunaNorbert) [inline src/App.svelte:231]: I think this should point to `DataPortal`, volume page

**Review by idiom-bytes — APPROVED** [review]
lgtm

**Pull request merged**